### ISSUE TITLE
Permissions bug. Document uploader unable to view their upload

### ISSUE THREAD

**Issue by pinarpink** [elixir-luxembourg/daisy#196]
**Describe the bug (current behaviour)**

A regular DAISY user (a researcher but not a PI) can create DAISY _Project_ records and attach _Documents_ to those records. Once upload is complete the user is not able to view the attachment just uploaded. 
This bug is reported by DLSM users but observable on all deployments..

**To reproduce**

1. Go to daisy.lcsb.uni.lu login with an account that has admin priviliges
2. Go to projects page
3. Search for 'ND Collection'
4. Go to ND Collection view page and click on the eye icon to see privileges. Screenshot also attached. In this setting those local custodians that are not a VIP do not have the document view privilege.

**Expected behavior**

All users in the local custodian list should have all rights on the record. Also based on experience with users we observe that fine-grained permission handling is not at all used and better be simplified. There is a separate issue for that 

**Environment:**

 - Deployment: both DLSM and LCSB deployments
 - Version DAISY v 1.4.0

 
**Additional information (error logs, screenshots)**
![Screenshot 2020-12-09 at 14 12 06](https://user-images.githubusercontent.com/2318344/101634660-4de73400-3a29-11eb-815d-bf4a76224384.png)
